cmd/run.go

```diff
@@ -455,9 +456,22 @@ a commandline interface for interacting with it.`,
 				Metrics: engine.Metrics,
 				Time:    engine.Executor.GetTime(),
 			}
-			// TODO: add json summary later
-			switch {
-			default:
+			if conf.ReportSummary.Valid {
+				var w io.Writer = stdout
+				if conf.ReportSummary.String != "" {
+					f, err := os.Create(conf.ReportSummary.String)
+					if err != nil {
+						panic(err)
```

Comment:
Panics are **NOT** a way to report problems to users.
 Not being able to open a file that we want to write to is no reason for the whole program to spew it's guts on the floor. A  simple log and return would be much better